source/index.txt

```diff
@@ -13,6 +13,7 @@
    :maxdepth: 1
```

Comment:
[q] Not part of this diff, but is there a section here to comment in that links to the new Connection pages?

Reply:
The `\connect` entry links to connect.txt. The links for the pages in the Connect section can be found in the connect.txt file: https://github.com/mongodb/docs-scala/pull/80/files#diff-b097e010c9e7918edece06cd4f7dc5e362cd66f61aa38035127990c2c4dac66e.